components/02-components/02-content-group/02-news-group/news-group.hbs

```diff
@@ -18,7 +18,7 @@
         <div class="col-xxl-4 col-xl-4 col-lg-4 col-md-6 col-sm-12">
             <div class="news-card-componet-div blue">
                 <div class="news-card-img">
-                    <img src={{ news-image }} alt="Card Image" />
+                    <img src="{{path '/college-dls/college/images/News_Blog.png'}}" alt="Card Image" />
```

Comment:
While the images show locally they don't show on remote repo. So updated images with {{path ... }} like you showed us originally.